package.json

```diff
@@ -12,9 +12,10 @@
   ],
   "license": "MIT",
   "type": "module",
-  "main": "dist/index.cjs",
-  "module": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "exports": "./dist/index.js",
+  "types": "./dist/index.d.ts",
```

Comment:
When `exports` is present, all other fields are ignored. You'll need to provide matchers for different environments:

```json
"exports": {
  "types": "./dist/index.d.ts",
  "require": "./dist/index.mjs",
  "import": "./dist/index.js"
},
```

`npx publint` or https://publint.dev can serve as a sanity check.

Reply:
thank you for checking, I'll fix this